micrometer-core/build.gradle

```diff
@@ -82,9 +82,8 @@ dependencies {
     }
 
     // Aspects
-    implementation libs.aspectjrt
-    java11Implementation libs.aspectjrt
-    optionalApi 'org.aspectj:aspectjweaver'
+    optionalApi libs.aspectjrt
+    java11RuntimeOnly libs.aspectjrt
```

Comment:
this still adds this as a transient dependency for the Java 11 build, is that OK?

Reply:
I don't think we need the dependency at all for the java11 sources in micrometer-core. There is no aspect-related code [there](https://github.com/micrometer-metrics/micrometer/tree/d162bddb4310df2136c5371e3be63cecc515b377/micrometer-core/src/main/java11/io/micrometer/core/instrument/binder/jdk), and the code that is there is deprecated in favor of the separate module [micrometer-java11](https://github.com/micrometer-metrics/micrometer/tree/2b5181cb384811e3f86d85685d6572f13f3ca481/micrometer-java11).

Reply:
If I do that then we get :disappointed: 

```
* What went wrong:
Execution failed for task ':micrometer-core:compileJava11Java'.
> Cannot infer AspectJ class path because no AspectJ Jar was found on class path: configuration ':micrometer-core:java11RuntimeClasspath'
```

Reply:
Maybe we should remove the whole multi jar config from micrometer-core ?